src/index.js

```diff
@@ -0,0 +1,119 @@
+"use strict";
+
+const state = {
+    city: 'San Francisco',
+    temperature: 75
+  };
+
+  const temperatureDisplay = document.getElementById('temperature');
+  const cityInput = document.getElementById('input_city');
+  const cityDisplay = document.getElementById('display_city');
+  const skyInput = document.getElementById('input_sky');
+
+  const displayTemperature = () => {
+    temperatureDisplay.textContent = `${state.temperature} °F`;
+  };
+
+  const increaseTemperature = () => {
+    state.temperature ++;
+    displayTemperature();
+    changeTemperatureDecor();
+  };
+
+  const decreaseTemperature = () => {
+    state.temperature --;
+    displayTemperature();
+    changeTemperatureDecor();
+  };
+
+  const changeTemperatureDecor = () => {
+    const landscape = document.getElementById('landscape');
+    if (state.temperature >= 80) {
+      temperatureDisplay.className = 'hot_color';
+      landscape.className = 'hot';
+    } else if (state.temperature >= 70 && state.temperature < 80) {
+      temperatureDisplay.className = 'warm_color';
+      landscape.className = 'warm';
+    } else if (state.temperature >= 60 && state.temperature < 70) {
+      temperatureDisplay.className = 'cool_color';
+      landscape.className = 'cool';
+    } else if (state.temperature >= 50 && state.temperature < 60) {
+      temperatureDisplay.className = 'chilly_color';
+      landscape.className = 'cold';
+    } else {
+      temperatureDisplay.className = 'cold_color';
+      landscape.className = 'cold';
+    }
+  };
+
+  const changeCity = () => {
+    state.city = cityInput.value;
+    cityDisplay.textContent = state.city;
+  };
+
+  const getRealtimeTemperature = async () => {
+    const locationData = await axios.get(
+      `http://127.0.0.1:5000/location?q=${state.city}`);
+    const lat = locationData.data[0].lat;
+    const lon = locationData.data[0].lon;
+    getWeatherByLocation(lat, lon);
+  };
+
+  const getWeatherByLocation = async (lat, lon) => {
+    const weatherData = await axios.get(
+      `http://127.0.0.1:5000/weather?lat=${lat}&lon=${lon}`
+    );
+    const temperatureKelvin = weatherData.data.current.temp;
+    state.temperature = convertKelvinToFahrenheit(temperatureKelvin)
+    displayTemperature();
+    changeTemperatureDecor();
+  };
```

Comment:
I would suggest adding some error handling to your functions. You could use a try/catch using async/await. Here is an example 
```javascript 
async function exampleFunction() {
  try {
    let response = await fetch('http://no-such-url');
  } catch(err) {
    alert(err); // TypeError: failed to fetch
  }
}
```
Here is a resource that goes in more in-depth: https://javascript.info/async-await